neurodiffeq/generators.py

```diff
@@ -7,20 +7,20 @@
 
 def _chebyshev_first(a, b, n):
     nodes = torch.cos(((torch.arange(n) + 0.5) / n) * np.pi)
-    nodes = ((a + b) + (b - a) * nodes) / 2
+    nodes = ((a + b) + (a - b) * nodes) / 2
```

Comment:
Is this change meant to ensure the returned points are increasing?

Reply:
Yes, that's the reason